### PR TITLE
DOC-1341: RS - Update LDAP docs to reflect changes

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial drafts

### CODE REVIEW

**Comment by lanceleonard** [comment]
Staged previews [are available](https://docs.redis.com/staging/jira-doc-1341/rs/security/ldap/).

File: content/rs/release-notes/rs-6-0-20-april-2021.md [hunk 1]
```diff
@@ -75,9 +74,9 @@ And other functional and stability improvements.
 #### New LDAP integration
 
 Redis Enterprise Software integrates Lightweight Directory Access Protocol
-(LDAP) authentication and authorization into its [role-based access controls]
+(LDAP) authentication and authorization into its [role-based access controls]({{<relref "rs/security/passwords-users-roles.md##olebased-access-control">}})
```
Comment (rrelledge) [inline content/rs/release-notes/rs-6-0-20-april-2021.md:77]:
```suggestion
(LDAP) authentication and authorization into its [role-based access controls]({{<relref "/rs/security/passwords-users-roles#rolebased-access-control">}})
```

File: content/rs/security/ldap/cluster-based-ldap-authentication.md [hunk 2]
```diff
@@ -12,13 +12,13 @@ aliases: [
 ]
 ---
 {{<warning>}}
-This article describes features that are considered deprecated.  They will be removed in a future release of Redis Enterprise Software.<br/><br/>  
+As of v6.2.12, the features described in this article are [obsolete]({{< relref "/glossary/#obsolete" >}}); that is, they have been removed from the product.<br/><br/>
```
Comment (rrelledge) [inline content/rs/security/ldap/cluster-based-ldap-authentication.md:15]:
```suggestion
As of v6.2.12, the features described in this article are [obsolete]({{< relref "/glossary/#obsolete" >}}) and have been removed from the product.<br/><br/>  
```

File: content/rs/security/ldap/migrate-to-role-based-ldap.md [hunk 3]
```diff
@@ -43,9 +39,9 @@ This checklist covers the basic process:
 
  Because deployments and requirements vary, you’ll likely need to adjust these guidelines.
 
-## Ways to test access
+## Test LDAP access
 
-There are several ways to test your LDAP integration, including:
+To test your LDAP integration, including:
```
Comment (rrelledge) [inline content/rs/security/ldap/migrate-to-role-based-ldap.md:44]:
```suggestion
To test your LDAP integration, you can:
```

File: content/rs/security/ldap/migrate-to-role-based-ldap.md [hunk 4]
```diff
@@ -43,9 +39,9 @@ This checklist covers the basic process:
...
 
 - Connecting with `redis-cli` and using the AUTH command to test LDAP username/password credentials.
```
Comment (rrelledge) [inline content/rs/security/ldap/migrate-to-role-based-ldap.md:46]: Maybe remove the -ing verbs in this list of test methods.
```suggestion
- Connect with `redis-cli` and use the `AUTH` command to test LDAP username/password credentials.
```

File: content/rs/security/ldap/migrate-to-role-based-ldap.md [hunk 5]
```diff
@@ -7,19 +7,15 @@ categories: ["RS"]
 aliases: 
 ---
 
-As of v6.0.20, Redis Enterprise Software supports two LDAP authentication mechanisms: the [cluster-based mechanism]({{< relref "/rs/security/ldap/cluster-based-ldap-authentication.md" >}}) supported in earlier versions and a [role-based mechanism]({{< relref "/rs/security/ldap/" >}}).
+Redis Enterprise Software supports LDAP through a [role-based mechanism]({{< relref "/rs/security/ldap/" >}}), first introduced [in v6.0.20]({{< relref "rs/release-notes/rs-6-0-20-april-2021" >}}).
 
-If you currently rely on the cluster-based mechanism, you can continue to do so in the short term.  However:
+(Support for a earlier, [cluster-based mechanism]({{< relref "/rs/security/ldap/cluster-based-ldap-authentication.md" >}}) was removed in v6.2.12.)
```
Comment (rrelledge) [inline content/rs/security/ldap/migrate-to-role-based-ldap.md:12]:
```suggestion
(Support for an earlier, [cluster-based mechanism]({{< relref "/rs/security/ldap/cluster-based-ldap-authentication.md" >}}) was removed in v6.2.12.)
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just left a couple of suggestions.

File: content/rs/security/ldap/migrate-to-role-based-ldap.md (hunk 5 above)
Comment (lanceleonard) [reply content/rs/security/ldap/migrate-to-role-based-ldap.md]: Fixed.

File: content/rs/security/ldap/migrate-to-role-based-ldap.md (hunk 4 above)
Comment (lanceleonard) [reply content/rs/security/ldap/migrate-to-role-based-ldap.md]: Fixed.

**Pull request merged**